src/TestUtils/GcloudWrapper/CloudFunction.php

```diff
@@ -118,17 +119,21 @@ public function deploy($options = [])
 
             return false;
         }
+
         $cmd = sprintf(
-            'gcloud -q %s%s deploy %s --entry-point %s --runtime %s --project %s --region %s %s --no-allow-unauthenticated',
+            'gcloud -q %s%s deploy %s --project %s --region %s %s --no-allow-unauthenticated%s',
```

Comment:
I take it the absence of a space between `unauthenticated` and `%s` is intentional?

Reply:
Correct. The injection of deployFlags there is preceded by a space.